robot/Cumulus/resources/NPSP.py

```diff
@@ -624,20 +624,13 @@ def check_related_list_values(self,list_name,*args):
             locator = npsp_lex_locators['check_related_list_item'].format(list_name,value)
             self.selenium.page_should_contain_element(locator)
 
-    def verify_eng_plan_exists(self,name, delete=None):  
-        """verifies that the Engagement Plans related list has a plan stored under it and clicks on dropdown if True is passed as 2nd argument"""
-        locator = npsp_lex_locators['record']['related']['link'].format('Engagement Plans',name)
-        self.selenium.page_should_contain_element(locator) 
-        plan=self.selenium.get_webelement(locator).text       
-        return plan
-    
-    
     def check_activity_tasks(self, *args):
         """verifies that the specified tasks are present under activity tab """
         for value in args:
             locator = npsp_lex_locators['engagement_plan']['tasks'].format(value)
             self.selenium.page_should_contain_element(locator)
 
+    @capture_screenshot_on_error
     def enter_level_values(self, **kwargs):
```

Comment:
as this keyword is specific to levels, plz move it to levels page object

Reply:
Moved all the engagement and level specific methods to the respective page objects